Dockerfile.deploy.runtime

```diff
@@ -0,0 +1,12 @@
+FROM fabric8/fabric8-openshift-nginx:vd83b3a1
+MAINTAINER "Pete Muir <pmuir@bleepbleep.org.uk>"
```

Comment:
Nope.
He's not gonna maintain this. Either add your name or remove this line. :P

Reply:
yes, copy-paster error!
thanks, removed in https://github.com/fabric8-ui/fabric8-planner/pull/2426/commits/e1c90ef1388ba15790c2dd04905c0d36e863e594